pkg/services/authz/zanzana.go

```diff
@@ -136,6 +136,16 @@ func (z *Zanzana) start(ctx context.Context) error {
 }
 
 func (z *Zanzana) running(ctx context.Context) error {
+	go func() {
+		if z.cfg.Zanzana.ListenHTTP {
```

Comment:
maybe we should add a cfg.Environment == settings.Dev check here so it's not accidentally started in prod envs